### PR TITLE
Adding -> Configuration_flava.py

### PR DESCRIPTION
**Pull request by AShreyam**
# What does this PR do?

Add configuration_flava.py to utils/documentation_tests.txt for doctest.

Based on issue https://github.com/huggingface/transformers/issues/19487

@sgugger could you please take a look at it?
Thanks =)

### CODE REVIEW

File: src/transformers/models/flava/configuration_flava.py [hunk 1]
```diff
@@ -16,7 +16,7 @@
 
 import copy
 import os
-from typing import Any, Dict, Union
+from typing import Any, Union, Dict
```
Comment (sgugger) [inline src/transformers/models/flava/configuration_flava.py:19]: Not sure why this change is in there but it should be reverted.

**Review by sgugger — COMMENTED** [review]
Thanks for your PR. Make sure to run `make style` on your branch for the quality tests to pass.

**Comment by ydshieh** [comment]
Hi @AShreyam 

This model config is done in #19724. Sorry if I forgot to follow this PR earlier. Going to close the PR though.
Thank you however.
